apidoc/Titanium/Contacts/Person.yml

```diff
@@ -160,15 +178,31 @@ properties:
         `string` type value.
         
         The `service` key value may be one of `AIM`, `Facebook`, `GaduGadu`, `GoogleTalk`, `ICQ`, 
-        `MSN`, `QQ`, `Skype`, or `Yahoo`.
+        `Jabber`, `MSN`, `QQ`, `Skype`, or `Yahoo`.
         
         On BlackBerry, the following apply: `aim`, `aliwangwang`, `bbmPin`, `googleTalk`, `icq`,
-        `irc`, `jabber`, `msLcs`, `msn`, `qq`, `sametime`, `skype`, `yahooMessenger`, `yahooMessenger`
+        `irc`, `jabber`, `msLcs`, `msn`, `qq`, `sametime`, `skype`, `yahooMessenger`
 
 
     type: Dictionary
     platforms: [android, iphone, ipad, blackberry]
     
+  - name: socialProfile
+    summary: |
+        Social profile information of the person. Multi-value.
+    description: |
+        The format of the dictionary accepted by this property is as follows:
+        
+        * Keys: any of `home`, `work` and/or `other`.
```

Comment:
I followed the same format as the existing instantMessage. Even the docs describes that these `keys` (`facebookProfile`,`flickrProfile`) have to be a subset of `home`, `work` and `other` keys. Not sure if it's because of achieving parity with Android. I do agree with you that there are predefined values though, will look into this further.